.github/workflows/update_file.yaml

```
@@ -0,0 +1,55 @@
+# This workflow is to sync dashboards and alert rules from the Sunbeam repo and create a PR if there is an update.
+name: Update files
+on:
+  workflow_dispatch:
+  workflow_call:
+  schedule:
+    - cron: '43 1 * * 1'
+
+
+env:
+  REPO_OWNER: canonical
+  REPO_PATH: openstack-exporter-operator
+
+jobs:
+
+  pull-from-sunbeam:
+    name: Pull files from sunbeam
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout local repository
+        uses: actions/checkout@v4
+
+      - name: set git config
+        env:
+          GH_TOKEN: ${{ github.token }}
+        run: |
+          git config --global user.email "${GITHUB_ACTOR_ID}+${GITHUB_ACTOR}@users.noreply.github.com"
+          git config --global user.name "$(gh api /users/${GITHUB_ACTOR} | jq .name -r)"
+          git config -l
+          
+      - name: Pull dashboard and alert rules from sunbeam
+        run: |
+          chmod +x scripts/pull_dashboard_and_alert_rules.sh
+          ./scripts/pull_dashboard_and_alert_rules.sh
+
+      - uses: actions/create-github-app-token@v1
+        id: app_token
+        with:
+          app-id: ${{ secrets.APP_ID }}
+          owner: ${{ env.REPO_OWNER }}
+          private-key: ${{ secrets.APP_PEM }}
+          repositories: ${{ env.REPO_PATH }}
+      - name: Create pull request
+        uses: canonical/create-pull-request@main
+        with:
+          upsert: true  # Allow update PRs
+          github-token: ${{ steps.app_token.outputs.token }}
+          branch-name: chore/auto-libs  # create new branch with the changes as the source branch for the PR
```

```suggestion
          branch-name: chore/sunbeam-sync  # create new branch with the changes as the source branch for the PR
```
iiuc we are not syncing any library